client/tokenized-express-checkout/event-handlers.js

```diff
@@ -3,59 +3,86 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { applyFilters } from '@wordpress/hooks';
+
 /**
  * Internal dependencies
  */
-import {
-	getErrorMessageFromNotice,
-	normalizeOrderData,
-	normalizePayForOrderData,
-	normalizeShippingAddress,
-	normalizeLineItems,
-	getExpressCheckoutData,
-} from './utils';
+import { getErrorMessageFromNotice, getExpressCheckoutData } from './utils';
 import {
 	trackExpressCheckoutButtonClick,
 	trackExpressCheckoutButtonLoad,
 } from './tracking';
+import ExpressCheckoutCartApi from './cart-api';
+import {
+	transformStripePaymentMethodForStoreApi,
+	transformStripeShippingAddressForStoreApi,
+} from './transformers/stripe-to-wc';
+import {
+	transformCartDataForDisplayItems,
+	transformCartDataForShippingOptions,
+	transformPrice,
+} from './transformers/wc-to-stripe';
```

Comment:
Used instead of the "normalize" functions from `'./utils` (they're the same `transform*` utilities that were part of the `tokenized-payment-request`).